docs/semgrep-appsec-platform/dashboard-beta.md

```diff
@@ -0,0 +1,135 @@
+---
+slug: dashboard-beta
+title: Dashboard (beta)
+hide_title: true
+description: Use the Semgrep dashboard (beta) to gain an overview of your organization's security posture, including the deployment of guardrails.
+tags:
+  - Semgrep AppSec Platform
+  - Secure guardrails
+---
+
+# Dashboard (beta)
+
+The Semgrep dashboard (beta) is an overview of your organization’s security posture from data aggregated within Semgrep AppSec Platform. It helps you achieve the following goals:
+
+- Evaluation of the success of your AppSec program, enabling you to know your current security risk.
+  - This includes assessing the deployment of secure guardrails to your organization.
+- Awareness of trends and opportunities that you can use to improve your security posture.
+  - You are able to view **recurring security issues**, consequently taking action on them.
+
+<!--
+The dashboard provides the following features:
+
+- Filters that enable you to track key metrics over a period of time, per Semgrep product, and per project.
+-->
+
+:::note
+This feature is in a **private beta**. To request access, contact your Technical Account Manager or your Account Executive and let them know you'd like to join the dashboard beta.
+:::
+
+
+![Dashboard (beta) page](/img/dashboard-fold.png)
+_**Figure**. The dashboard (beta_) page.
+
+The dashboard is divided into several panes:
```

Comment:
i think this is prob a good opportunity for us to add some color on why these charts are important / what they help folks answer? cc @milanwilliams

Reply:
some thoughts (mostly pulled from a slack post from milan, but would use her opinion as source of truth!):

**Production backlog:** 
This section can help answer the following questions: 
- How is my security posture doing over time? 
- Is my backlog decreasing or increasing? 
- Is my net change increasing/decreasing? 
- Are we addressing findings faster than new findings are coming in? 

**Secure guardrails:**
This section can help answer the following questions: 
- How many vulnerabilities did Semgrep prevent from hitting production, over time?
- Am I effectively introducing guardrails to my developers?  
- Of the issues shown to developers, are things being fixed or are they being ignored? 

**Most findings by project:**
This section can help answer the following questions: 
- Which of my projects have the most findings in a particular product area?
- Which of my projects have the most findings for a particular severity?

 **Median open finding age:**
This section can help answer the following question: 
- What is the amount of time a finding remains open, by product or by severity?

cc @milanwilliams

Reply:
This is super helpful and I believe the money shot for the doc. Thank you @sims-e